### PR TITLE
Cleaned up warnings

### PR DESCRIPTION
**Pull request by rawkode**
Implemented single_tag test to remove warning and addded _outline to remove unused variable warning

### CODE REVIEW

File: lib/gherkin/gherkin.ex [hunk 1]
```diff
@@ -72,7 +72,7 @@ defmodule Gherkin do
       outline = %Gherkin.Elements.ScenarioOutline{}
       Gherkin.scenarios_for(outline) |> Enum.each(&run_scenario/1)
   """
-  def scenarios_for(outline = %Elements.ScenarioOutline{name: name, tags: tags, steps: steps, examples: examples}) do
+  def scenarios_for(_outline = %Elements.ScenarioOutline{name: name, tags: tags, steps: steps, examples: examples}) do
```
Comment (mgwidmann) [inline lib/gherkin/gherkin.ex]: If we aren't using it (perhaps at one time between commits it was being used 😆 ), then we can just remove the variable as the struct is descriptive enough, dont you think?

File: test/gherkin/parser_test.exs [hunk 2]
```diff
@@ -50,6 +50,17 @@ defmodule Gherkin.ParserTest do
     Given there are 1 coffees left in the machine
   """
 
+  @feature_with_multiple_feature_tag """
+  @beverage @coffee
```
Comment (mgwidmann) [inline test/gherkin/parser_test.exs:55]: I noticed this also doesn't support multiple lines of tags like:

```
@beverage
@coffee
Feature: Serve coffee
  ...
```

But that can be addressed later if you're not up to it

Forgot that I already added an issue for that #4 😆

**Review by mgwidmann — APPROVED** [review]

File: test/gherkin/parser_test.exs (hunk 2 above)
Comment (rawkode) [reply test/gherkin/parser_test.exs:55]: I'll take a look at it 😃

**Comment by rawkode** [comment]
@mgwidmann Updated to support tags across multiple lines and removed unused variable

:+1:

**Comment by rawkode** [comment]
Rebased from upstreeam

**Comment by mgwidmann** [comment]
This looks great! Thanks!

**Pull request merged**